app/assets/stylesheets/frontend/print/_documents-index.scss

```diff
@@ -4,26 +4,24 @@
   }
 
   .filter-results-block {
-    .inner-block {
-      .feeds {
-        display: none;
-      }
+    .feeds {
+      display: none;
+    }
 
-      ol,
-      ul {
-        padding: 0;
-        margin: 0;
+    ol,
```

Comment:
Wondering if list styling is needed - can this be used? https://components.publishing.service.gov.uk/component-guide/list

Reply:
I think this applies to print style and is currently used. I'd say we should be able to remove the styles when we swap in the "document list" component though so will add a note in Trello to do that.